be/src/runtime/fragment_mgr.cpp

```diff
@@ -447,7 +446,7 @@ Status FragmentMgr::exec_plan_fragment(const TExecPlanFragmentParams& params, co
         }
         return Status::InternalError(error_msg);
     } else {
-        start_cb((*exec_state_ptr)->executor());
+        start_cb(exec_state->executor());
```

Comment:
I suggest that you'd better to add comment for your change to avoid somebody revert your change.
Because without more comment, people can't get it.

Reply:
Fixed